client/commands/v2/pysa_server.py

```diff
@@ -102,15 +169,22 @@ async def process_open_request(
         self, parameters: lsp.DidOpenTextDocumentParameters
     ) -> None:
         document_path = parameters.text_document.document_uri().to_file_path()
+        self.file_tracker[str(document_path)] = True
```

Comment:
Any time you're only storing `True` in a dict, that's a good sign that you probably don't actually need a `dict`.  Instead, if you just want to see if something is in a collection or not (and maintain uniqueness), you could be using a `set`.